docs/docs/overview/quickstart/get-started-with-angular.md

```diff
@@ -0,0 +1,419 @@
+- **Editor** - You can add text that you want displayed in each notification item. Additionally, you can specify custom variables using `{{ }}`. This means you can inject variables from your code into a notification item's text via a payload.
+
+1. Feel free to add only text for now and rename the notification template to `Onboarding In App`. It automatically creates a slug-like identifier that will be needed in later steps to trigger a notification.
+
+![Screenshot 2023-05-21 at 09.33.43.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776584/Screenshot_2023-05-21_at_09.33.43_ujx2td.png)
+
+![Screenshot 2023-05-21 at 09.20.51.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776585/Screenshot_2023-05-21_at_09.20.51_cueewk.png)
+
+Next, we’ll learn how to create subscribers on Novu - _Recipients of Notifications_
+
+## Create A Subscriber
+
+Click **Subscribers** on the left sidebar of the [Novu dashboard](https://web.novu.co/subscribers) to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign up.
+
+![subscribers.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776584/subscribers_ppir3w.png)
+
+Now, let's create a subscriber on Novu.
+
+Novu has a plethora of backend SDKs (Node.js, PHP, .NET, Go, Ruby, Python and Kotlin) to choose from to create a subscriber programmatically. This is the recommended method.
+
+<iframe width="800" height="450" src="https://codesandbox.io/p/sandbox/create-subscriber-m5uibf?file=%2Findex.js%3A1%2C1&embed=1" allowfullscreen></iframe>
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.identify('132', {
+  email: 'john.doe@domain.com',
+  firstName: 'John',
+  lastName: 'Doe',
+  phone: '+13603963366',
+});
+```
+
+<iframe width="800" height="450" src="[https://codesandbox.io/p/sandbox/clever-matan-m5uibf?embed=1](https://codesandbox.io/p/sandbox/clever-matan-m5uibf?embed=1)" allowfullscreen></iframe>
+
+Obtain your API key from your [Novu dashboard.](https://web.novu.co/settings) Replace `YOUR_NOVU_API_KEY_HERE` with it.
+
+Now check your Novu dashboard. You should see the recently created subscriber.
+
+You can also update the subscriber info like so:
+
+<iframe width="800" height="450" src="https://codesandbox.io/p/sandbox/create-subscriber-forked-r625n3?file=%2Findex.js%3A21%2C1&embed=1" allowfullscreen></iframe>
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu('<YOUR_NOVU_API_KEY>');
+
+await novu.subscribers.update('132', {
+  email: 'janedoe@domain.com', // new email
+  phone: '+19874567832', // new phone
+});
+```
+
```

Comment:
Please use the multi-language component so that you can add the code for updating subscribers in PHP as well.